lib/actions-oclif/env/rm.ts

```diff
@@ -91,17 +92,28 @@ export default class EnvRmCmd extends Command {
 		await Command.checkLoggedIn();
 
 		const { confirm } = await import('../../utils/patterns');
+		const ids = params.id.toString().split(',');
 		await confirm(
 			opt.yes || false,
-			'Are you sure you want to delete the environment variable?',
+			ids.length > 1
+				? `Are you sure you want to delete ${ids.length} environment variables?`
+				: `Are you sure you want to delete environment variable ${ids[0]}?`,
 			undefined,
 			true,
 		);
 
 		const balena = getBalenaSdk();
-		await balena.pine.delete({
-			resource: ec.getVarResourceName(opt.config, opt.device, opt.service),
-			id: params.id,
-		});
+		for (const id of ids) {
+			try {
+				await balena.pine.delete({
+					resource: ec.getVarResourceName(opt.config, opt.device, opt.service),
+					id: parseInt(id, 10),
+				});
+			} catch (err) {
+				console.info(`${err.message}, id: ${id}`);
```

Comment:
`console.error` instead of `console.info` (so it goes to `stderr` instead of `stdout`). I think it should have been `console.error` in `lib/actions-oclif/device/move.ts` as well (you can change it in this PR too, optionally in a separate commit; the CHANGELOG file will list each commit message as a separate bullet point, under the same new CLI version).

Arguably, informational messages should go to stderr as well:
```
    console.info(`${uuid} was moved to ${application}`);
```
But the case for info messages is less clear, especially when the CLI command is not expected to "print an answer". For a different, clearer (?) example:
```
$ ./compute-ultimate-answer
Computing ultimate answer...
Still computing...
42
```
In the output above, `Computing ultimate answer...` and `Still computing...` would go to stderr (even though they are not error messages), and `42` would go to stdout. The balena CLI is far from being consistent in this regard, though...

What is less controversial (I think) is that an error message for an error that warrants `process.exitCode = 1` should go to stderr, i.e. `console.error`.

Reply:
> Arguably, informational messages should go to stderr as well:

    console.info(`${uuid} was moved to ${application}`);

I changed the other error messages to `console.error()` as I think they are pretty clearly errors. I left this as `console.info()` as I think it informative rather than an error. I would say this message should be removed and only provide output during an error if anything.